dataland-frontend/src/components/forms/CreateEuTaxonomyForFinancials.vue

```diff
@@ -56,9 +56,10 @@
               <FormKit type="group" name="data" label="data" validation-label="data" validation="required">
                 <UploadReports
                   ref="UploadReports"
+                  :isEuTaxonomy="true"
                   :editMode="editMode"
                   :referencedReportsForPrefill="templateDataset?.referencedReports"
-                  @referenceableReportNamesChanged="handleChangeOfReferenceableReportNames"
+                  @documentUpdated="handleChangeOfReferenceableReportNames"
```

Comment:
Why has the name of this event changed?
I personally prefer the name as it was before because it was clearer.

Reply:
To align with other components which have the same event name.

Reply:
But it violates our naming convention for reports/documents/files:

We have had a team discussion some time ago and have agreed that a "document to upload" should always be a file accompanied by the hash-reference.
A report on the other hand, only contains the the file. See the types in `FileUploadUtils.ts`:

![image](https://github.com/d-fine/Dataland/assets/99255296/65ee59ad-1cca-43cf-a7e7-359eba9a9b0f)


Now your name change violates this, since it says "documentUpdated", while it is actually only reports that are being updated.

Reply:
Has been fixed in other conversation. Closing.